internal/datastore/postgres/postgres.go

```diff
@@ -373,16 +377,43 @@ func (pgd *pgDatastore) Features(ctx context.Context) (*datastore.Features, erro
 
 func buildLivingObjectFilterForRevision(revision datastore.Revision) queryFilterer {
 	return func(original sq.SelectBuilder) sq.SelectBuilder {
-		return original.Where(sq.LtOrEq{colCreatedTxn: transactionFromRevision(revision)}).
-			Where(sq.Or{
-				sq.Eq{colDeletedTxn: liveDeletedTxnID},
-				sq.Gt{colDeletedTxn: revision},
-			})
+		txID := transactionFromRevision(revision)
+
+		createdBeforeTXN := sq.Expr(fmt.Sprintf(
+			snapshotAlive,
+			colCreatedXid,
+			colSnapshot,
+			tableTransaction,
+			colXID,
+			sq.Placeholders(1),
```

Comment:
What does the `1` mean in `Placeholders`?

Reply:
1 placeholder. Basically `?`, if you said 3 it would be `?, ?, ?`